modules/renter/files_test.go

```diff
@@ -125,23 +125,21 @@ func TestRenterDeleteFile(t *testing.T) {
 		t.Error("file was deleted, but is still reported in FileList")
 	}
 
-	/*
-		// Put a file in the renter, then rename it.
-		rt.renter.files["1"] = &file{
-			name: "one",
-		}
-		rt.renter.RenameFile("1", "one")
-		// Call delete on the previous name.
-		err = rt.renter.DeleteFile("1")
-		if err != ErrUnknownNickname {
-			t.Error("Expected ErrUnknownNickname:", err)
-		}
-		// Call delete on the new name.
-		err = rt.renter.DeleteFile("one")
-		if err != nil {
-			t.Error(err)
-		}
-	*/
+	// Put a file in the renter, then rename it.
+	f := newTestingFile()
+	f.name = "1"
+	rt.renter.files[f.name] = f
+	rt.renter.RenameFile(f.name, "one")
+	// Call delete on the previous name.
+	err = rt.renter.DeleteFile("1")
+	if err != ErrUnknownNickname {
+		t.Error("Expected ErrUnknownNickname, got", err)
+	}
+	// Call delete on the new name.
+	err = rt.renter.DeleteFile("one")
+	if err != nil {
+		t.Error(err)
+	}
```

Comment:
Need to add a check here that the disk operations correctly cleared the .sia files.